training/args.py

```diff
@@ -80,6 +80,18 @@ def _get_dataset_args(parser: argparse.ArgumentParser) -> None:
         default=[49],
         help="CogVideoX1.5 need to guarantee that ((num_frames - 1) // self.vae_scale_factor_temporal + 1) % patch_size_t == 0, such as 53"
     )
+    parser.add_argument(
+        "--base_height",
+        type=int,
+        default=480,
+        help="Base height for rotary positional embeddings",
+    )
```

Comment:
Could we use the value from transformer config instead of making this a parameter? base_height could be something like `model_config.sample_height * 8`, and likewise for width. This is because it's not easy for users to know beforehand what to specify here. The 8 comes from vae spatial compression ratio so that could be made a separate variable for clarity if you'd like